app/controllers/application_controller.rb

```diff
@@ -29,16 +29,17 @@ class ApplicationController < ActionController::Base
     )
   end
 
-  PUBLIC_CONTROLLERS = %w[shell
-                          async_info
-                          ga_events
-                          service_worker
-                          omniauth_callbacks
-                          registrations
+  PUBLIC_CONTROLLERS = %w[async_info
                           confirmations
+                          deep_links
+                          ga_events
+                          health_checks
                           invitations
+                          omniauth_callbacks
                           passwords
-                          health_checks].freeze
+                          registrations
+                          service_worker
+                          shell].freeze
```

Comment:
I reordered them alphabetically, not sure if they were ordered that way for a reason but I can always revert these changes. 

The only new entry here is `deep_links`